src/postgres/src/bin/pg_dump/pg_dump.c

```diff
@@ -1019,6 +1019,7 @@ help(const char *progname)
 	printf(_("  -c, --clean                  clean (drop) database objects before recreating\n"));
 	printf(_("  -C, --create                 include commands to create database in dump\n"));
 	printf(_("  -E, --encoding=ENCODING      dump the data in encoding ENCODING\n"));
+	printf(_("  -m, --masters=HOST:PORT      comma-separated list of YB-Master hosts and ports\n"));
```

Comment:
wait, it looks like this is already below (around line 1070 below this line). I think we can just modify the text there rather than add a newline.

Reply:
@nvcnvn last review comment here...

Reply:
resolved,
haha sorry for the noise, I should not try to much without paying attention that YB team already add the documents to the command.